keel-core/src/main/kotlin/com/netflix/spinnaker/keel/plugin/ResourceSpecExportHelper.kt

```diff
@@ -60,3 +61,13 @@ inline fun <reified T : Any, reified S : Any> buildSpecFromDiff(
     ctor.callBy(params)
   }
 }
+
+/**
+ * Helper function to return an instance of any type with all nullable constructor parameters set to null.
+ * Note that this call will fail if any remaining parameters to the primary constructor are not optional.
+ */
+fun <T : Any> emptyShell(klass: KClass<T>): T {
```

Comment:
Oops, this is a left-over from my attempt to fix the specifics of nullifying only some fields of `ClusterDeployStrategy`. Will remove.